tools/build-kani/src/sysroot.rs

```diff
@@ -106,7 +106,12 @@ pub fn build_lib() {
 
     // Collect the build artifacts.
     let artifacts = build_artifacts(&mut cmd);
-    let _ = cmd.wait().expect("Couldn't get cargo's exit status");
+    let exit_status = cmd.wait().expect("Couldn't get cargo's exit status");
+    // exit_ok is an experimental API where we could do .exit_ok().expect("Build failed")
+    // Until it is stable:
+    if !exit_status.success() {
+        panic!("Build failed");
```

Comment:
Can you please extend this message as:
```suggestion
        panic!("Build failed because `cargo build` didn't return a successful exit code");
```
or something similar. This information may help in debugging build issues.

Reply:
Thanks, done.